dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/HVDC/ACEmulationOuterControl.mo

```diff
@@ -0,0 +1,63 @@
+within Dynawo.Electrical.Controls.HVDC;
+
+/*
+* Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model ACEmulationOuterControl "AC Emulation outer control for HVDC"
+
+  Connectors.ImPin PRefSet1RawPu (value(start = PRefSet10Pu)) "Raw reference active power of HVDC link 1 in p.u (base SnRef)";
```

Comment:
Why defining 4 raw connectors and not define the outputs with the parameters only?

Reply:
Mainly to be able to connect a step (or something else) and change PRef or K. Maybe it is not relevant for K (for PRef too ?) and we can only keep a parameter for it. What do you think ?

Reply:
I changed KACEmulation to simple parameters